### DISCUSSION TITLE
Missing Metadata in Response , While Using Amazon Bedrock in Semantic Kernel

### DISCUSSION THREAD

**Discussion by sp-shukla** [microsoft/semantic-kernel#9908] [Q&A]
![image](https://github.com/user-attachments/assets/3f791d12-914b-4932-b4b4-9d5bffbe1c11)
I am using Amazon Bedrock with the Semantic Kernel framework to use chat completions. I am calling the GetChatMessageContentAsync  method  of IChatCompletionService to get  the response from the model.

The issue I am facing is that the response only contains the Content property (the message text). Other properties like metadata (e.g., tokens, model information, or other related fields) are missing in the response.

Is there a way to retrieve these additional properties, or am I missing a configuration step to enable the complete response data?

**Comment by sophialagerkranspandey** [comment]
Tagging @TaoChenOSU

**Comment by moonbox3** [comment]
This is the .Net SDK, so we'd want @RogerBarret0 or @markwallace-microsoft to help look into it.

**Comment by markwallace-microsoft** [comment]
Created an issue for this: https://github.com/microsoft/semantic-kernel/issues/9989
